cmd/endpoint.go

```diff
@@ -182,12 +182,26 @@ func (endpoints EndpointList) SetHTTP() {
 func NewEndpointList(args ...string) (endpoints EndpointList, err error) {
 	// isValidDistribution - checks whether given count is a valid distribution for erasure coding.
 	isValidDistribution := func(count int) bool {
-		return (count >= 4 && count <= 16 && count%2 == 0)
+		return (count >= minErasureBlocks && count <= maxErasureBlocks && count%2 == 0)
+	}
+
+	// getValidErasureBlocks returns a string containing the valid values for erasure coding.
+	getValidErasureBlocks := func() string {
+		list := ""
+		for blocks := minErasureBlocks; blocks <= maxErasureBlocks; blocks += 2 {
+			list += fmt.Sprintf("%d", blocks)
+			if blocks < maxErasureBlocks-2 {
+				list += ", "
+			} else if blocks == maxErasureBlocks-2 {
+				list += " or "
+			}
+		}
+		return list
```

Comment:
This way it would be more expensive as the slice is always computed whereas now it is only done in the case of error.